manager/integration/tests/test_csi.py

```diff
@@ -290,20 +291,55 @@ def test_csi_encrypted_block_volume(client, core_api, storage_class, crypto_secr
     storage_class['parameters']['csi.storage.k8s.io/node-publish-secret-namespace'] = LONGHORN_NAMESPACE  # NOQA
     storage_class['parameters']['csi.storage.k8s.io/node-stage-secret-name'] = 'longhorn-crypto'  # NOQA
     storage_class['parameters']['csi.storage.k8s.io/node-stage-secret-namespace'] = LONGHORN_NAMESPACE  # NOQA
+    storage_class['parameters']['encrypted'] = 'true'
     create_storage_class(storage_class)
 
     create_and_verify_block_volume(client, core_api, storage_class, pvc,
-                                   pod_manifest)
+                                   pod_manifest, False)
 
 
-def create_and_verify_block_volume(client, core_api, storage_class, pvc, pod_manifest):  # NOQA
+@pytest.mark.csi  # NOQA
+def test_csi_encrypted_migratable_block_volume(client, core_api, storage_class, crypto_secret, pvc, pod_manifest):  # NOQA
+    """
+    Test CSI feature: encrypted migratable block volume
```

Comment:
Suggest add an issue link for context referencing.

Reply:
Added. Thanks.